src/main/java/sparksoniq/spark/closures/ReturnFlatMapClosure.java

```diff
@@ -43,6 +44,7 @@ public ReturnFlatMapClosure(RuntimeIterator expression) {
         while (_expression.hasNext())
             result.add(_expression.next());
         _expression.close();
+        result.removeIf(Objects::isNull);
```

Comment:
I understand the temptation to do this, but this is risky -- if there are nulls (which there should never be), you will no longer know: it is a bit like putting the dust under the carpet or switching off the fire alarm rather than fighting the fire :-) It is better practice to let the exception be thrown so that you can fix all the places were a Java null lands in here (that way, you will catch all iterators in which a move of code from next() to open() is necessary).